packages/co-busboy/src/index.ts

```
@@ -0,0 +1,373 @@
+    file.on('end', onFileCleanup);
+    file.on('close', onFileCleanup);
+
+    const { filename, encoding, mimeType } = info;
+
+    if (checkFile) {
+      const err = checkFile(fieldname, file, filename, encoding, mimeType);
+      if (err) {
+        // Make sure request stream's data has been read
+        const blackHoleStream = new BlackHoleStream();
+        file.pipe(blackHoleStream);
+        return onError(err);
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)